azext_iot/central/params.py

```diff
@@ -104,6 +103,12 @@ def load_central_arguments(self, _):
             " Specify an ID that you'll then use when modifying or deleting this token later via the CLI or API.",
         )
         context.argument("role", arg_type=role_type)
+        context.argument(
+            "org_id",
+            options_list=["--organization-id", "--orgid"],
```

Comment:
The short alias should be `--org-id` to meet the conventions of existing args here are examples from this PR

![image](https://user-images.githubusercontent.com/28658112/133491405-357cbddf-30e6-41bf-91f7-e66487e13e3a.png)

Reply:
fixed